ContactDetailsApi/V1/Gateways/DynamoDbGateway.cs

```diff
@@ -44,5 +44,15 @@ public async Task<List<ContactDetails>> GetContactDetailsByTargetId(ContactQuery
 
             return contactDetailsEntities.ToDomain();
         }
+
+        [LogCall]
+        public async Task<ContactDetails> CreateContact(ContactDetailsRequestObject requestObject)
+        {
+            var personDbEntity = requestObject.ToDatabase();
```

Comment:
Surely a `contact` not a `personDbEntity`?

Reply:
Surely !